server/service/src/backend_plugin/boajs/methods/get_store_preferences.rs

```diff
@@ -0,0 +1,32 @@
+use boa_engine::*;
+
+use crate::{
+    backend_plugin::{boajs::utils::*, plugin_provider::PluginContext},
+    store_preference::get_store_preferences,
+};
+
+pub(crate) fn bind_method(context: &mut Context) -> Result<(), JsError> {
+    context.register_global_callable(
+        JsString::from("get_store_preferences"),
+        0,
+        NativeFunction::from_copy_closure(move |_, args, ctx| {
+            // TODO Is this actually safe ? (need to check reference counts after plugin has run)
```

Comment:
any updates?

Reply:
I've checked here: https://github.com/andreievg/Checking-global-static-context-in-boajs-callback

And updated to be scoped + the comments: b5483e59f16b07d6a5c504410f6b20a5cf23a2cd (in top of stack)